### PR TITLE
Adding React Native Expo Demo with Appwrite

### PR DESCRIPTION
**Pull request by Neeleshrj**
### I made an application to organize your medicines using Appwrite

* Related to this <a href="https://github.com/appwrite/appwrite/issues/4253#issuecomment-1277079086">issue</a>
* I will be soon writing a blog and publishing on dev.to

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -88,6 +88,7 @@ Showcase for projects built during [#30DaysOfAppwrite](https://30days.appwrite.i
 * [Flutter Appwrite Todo](https://github.com/devkishor8007/todo-with-flutter/tree/todo_appwrite/flutter_appwrite)
 * [KnowYourFood](https://github.com/LeBraveLittleToaster/KnowYourFood)
 * [Budgetist](https://github.com/intel-raja/budgetist_kakeibo#budgetist_kakeibo)
+* [Expo Appwrite Store your Medicines](https://github.com/Neeleshrj/store-your-meds.git)
```
Comment (stnguyen90) [inline README.md:91]: I don't think this should be part of the 30 days of Appwrite challenge. Would you please move it to the React section?

**Review by stnguyen90 — COMMENTED** [review]
Any chance you can add demo to your readme via [snack](https://snack.expo.dev/) or something? For example, there's an [Expo demo](https://expo.dev/@rne_org/react-native-elements) and [Snack](https://snack.expo.dev/@arpitbhalla/react-native-elements) on [this](https://github.com/react-native-elements/react-native-elements) Readme.

**Comment by PineappleIOnic** [comment]
Thank you so much for the PR 🤩. We're adding the `hacktoberfest-accepted` label to ensure this PR counts towards your Hacktoberfest contributions count. With that said, please stay active on this PR to address any comments once you receive a review. Happy Hacktoberfest! 🎃

**Comment by Neeleshrj** [comment]
> Any chance you can add demo to your readme via [snack](https://snack.expo.dev/) or something? For example, there's an [Expo demo](https://expo.dev/@rne_org/react-native-elements) and [Snack](https://snack.expo.dev/@arpitbhalla/react-native-elements) on [this](https://github.com/react-native-elements/react-native-elements) Readme.

Sure!

**Comment by stnguyen90** [comment]
@Neeleshrj have you had a chance to add a snack demo? Please make sure to mark the PR ready for review and re-request a review from me once you've added the demo.

**Comment by christyjacob4** [comment]
@Neeleshrj let me know if you're still working on it

**Comment by Neeleshrj** [comment]
Hi, no I am not working on it right now. I was not able to produce a snack demo and got busy with other stuff.

**Review by christyjacob4 — APPROVED** [review]

**Comment by christyjacob4** [comment]
THANK YOU! All changes merged 🥳

Please reach out to me on our [Discord server](https://appwrite.io/discord) if you would like to claim your Appwrite swags! As a way of saying thank you, we would also love to invite you to join the Appwrite organization on GitHub. Please share your GitHub username with us on Discord.  

You can accept the invite by visiting https://github.com/orgs/appwrite/invitation. By joining our team, you will officially be an Appwrite maintainer on GitHub.

You can change your membership [visibility settings](https://help.github.com/en/github/setting-up-and-managing-your-github-user-account/publicizing-or-hiding-organization-membership), so your new Appwrite team membership badge will show up on your personal GitHub profile.

Please feel free to look for more PRs you might be interested in helping with on our long list of [Hacktoberfest friendly issues](https://github.com/search?q=org%3Aappwrite+org%3Autopia-php+org%3Aopen-runtimes+label%3Ahacktoberfest+created%3A%3E2021-01-01+state%3Aopen&type=issues&s=updated&o=asc) and help make Appwrite better :)

**Pull request merged**